docs/features/create_mesos_cluster.rst

```diff
@@ -59,15 +59,14 @@ Step-by-step Guide
   In order to enable brocker installation process variable enable_client_install should be set to "true".
   In case of "false" value, no brockers will be installed.
 
-  In order to repair kafka-mesos-scheduler application, variable repair must be set to "true". Default value is "false"
-
-  WARNING:: When repair is set to "true", all existing brockers are destroyed and kafka-mesos-scheduler application
-            is removed from Marathon.
-
 4. The next step is to run the main playbook with security file generated before::
 
         ansible-playbook -i inventory/<inventory file> site.yml -e @security.yml
 
+   If you need to repair an existing deployment, then run::
+
```

Comment:
I would like to paraphrase this sentence.
----
 When repair of existing cluster is required, the same play can be performed in addition with "repair_mode" set to "true".

Simply run::

bla-bla-lba

This variable will safely restart existing services.